public/code/person_id_tracker.py

```diff
@@ -0,0 +1,79 @@
+# python script to turn parentID into hierarchicall finesse
+# Made by Decruw Cedric and marc test
+
+# import 
+
+import csv
+import pandas as pd
+# dependencies
+
+path_file = "C:\\Users\\cedric\\Desktop\\new_excel_file_parentID.txt"
```

Comment:
Hardcoded local files?

Reply:
Jup the best variables. This is also a file that is used for testing the preview functionality of the webcomponent.
But I agree that using local variables is a very bad practice.